src/source/PeerConnection/SessionDescription.h

```diff
@@ -64,6 +64,7 @@ STATUS populateSessionDescription(PKvsPeerConnection, PSessionDescription, PSess
 STATUS reorderTransceiverByRemoteDescription(PKvsPeerConnection, PSessionDescription);
 STATUS setReceiversSsrc(PSessionDescription, PDoubleList);
 PCHAR fmtpForPayloadType(UINT64, PSessionDescription);
+INT32 getH264FmtpScore(PCHAR);
```

Comment:
UINT32

Reply:
:+1: Changing to DOUBLE per earlier discussion.